docs/README.md

```diff
@@ -0,0 +1,25 @@
+<p>
+    <img src="assets/images/aws_load_balancer_icon.svg" alt="AWS LoadBalancer Logo" width="200" />
+</p>
+
+## AWS LoadBalancer Controller
+
+AWS LoadBalancer Controller is a controller to help manage Elastic Load Balancers for a Kubernetes cluster.
+
+  - It satisfies Kubernetes [Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/) by provisioning [Application Load Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+  - It satisfies Kubernetes [Service resources](https://kubernetes.io/docs/concepts/services-networking/service/) by provisioning
+[Network Load Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html).
+
+This project was priorly known as "AWS ALB Ingress Controller", we rebranded it to be "AWS LoadBalancer Controller".
```

Comment:
formerly instead of priorly sounds better to me

Reply:
nice :D